### PR TITLE
Add version number

### PR DESCRIPTION
**Pull request by wbt**
Adds version number in a place where people can easily find it while or after Ganache is running.  See also https://github.com/trufflesuite/ganache/commit/d4b1cbd8776153a53d3e58d4339baedbad94415d

Ganache could benefit from a user-friendly way to know what version is running, from the GUI, and to be able to reconstruct that information when analyzing logs historically.  That the default download filename has no version number and the source link (AWS) strips that information out of the URL doesn't help.  (It is displayed briefly on startup in a splash screen).  

An easy way to do this is adding the version number to the startup line at the top of the logs, which is the intention of this contribution.

### CODE REVIEW

**Comment by benjamincburns** [comment]
Thanks for the contribution, @wbt! Per #241, I think the best way to accomplish this is to have an "about" page somewhere.

In the mean time I think that adding the version to the top of the logs is a great idea, but I'm not sure that all `console.log` output shows up on the logs page. IIRC, the logs page is filled with the output from the underlying instance of `ganache-cli` which `ganache` runs in the background. I'll give this patch a try however, just to make sure that I'm not mistaken.

File: src/chain.js [hunk 1]
```diff
@@ -3,6 +3,8 @@ var ganacheLib = require("ganache-cli")
 var path = require("path")
 var Web3 = require("web3")
 
+import * as pkg from '../../../package.json'
```
Comment (benjamincburns) [inline src/chain.js]: When I run `npm start` on this branch, startup hangs. When I check the logs, I get: `Error: Blockchain process exited prematurely with code '1', due to signal 'null'.`

After adding some extra logging to dump errors from the forked subprocess, I get the following:

```
SyntaxError: Unexpected token import
    at Object.exports.runInThisContext (vm.js:78:16)
    at Module._compile (module.js:543:28)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.runMain (module.js:605:10)
    at run (bootstrap_node.js:424:7)
    at startup (bootstrap_node.js:147:9)
    at bootstrap_node.js:544:3
```

**Comment by wbt** [comment]
I agree with #241 but having the version number in the logs helps in retroactive debugging too.  I figured putting it there would be the easiest option to address the core of #241 without requiring new UI (and still have the information easy to access, scrolling up to the top of the logs section).  

I've got a new commit above which will hopefully help on the error.

**Comment by benjamincburns** [comment]
I think this will still be broken. You likely want to drop the `import` and use `process.env.npm_package_version`, per https://docs.npmjs.com/misc/scripts#packagejson-vars

**Comment by seesemichaelj** [comment]
Hey @wbt, due to the large changes since January, I'm just going to make a new branch with these proposed changes. If you'd like to be a co-author the commit (as not to steal your thunder haha!) you can send it to me at `seese` `at` `trufflesuite.com`. You can find out your email:
```
To find your GitHub-provided no-reply email, navigate to your email settings page under "Keep my email address private."
```

**Comment by seesemichaelj** [comment]
closing in favor of #768

**Comment by wbt** [comment]
Thanks for merging in the functionality!

**Pull request closed without merging**